crates/api_common/src/context.rs

```diff
@@ -23,12 +25,23 @@ impl LemmyContext {
     client: ClientWithMiddleware,
     secret: Secret,
     rate_limit_cell: RateLimitCell,
+  ) -> LemmyContext {
+    let email_sender = Arc::new(DefaultEmailSender {});
```

Comment:
Not needed.

Reply:
Discussed in https://github.com/LemmyNet/lemmy/pull/3742#discussion_r1282285277 for the overall test strategy